frontend/i18next.scanner/en/translation.json

```diff
@@ -137,37 +138,37 @@
   "Reset password": "Reset password",
   "Confirm password": "Confirm password",
   "Confirm": "Confirm",
-  "We are going to send you an email containing a link to reset your password": "We are going to send you an email containing a link to reset your password",
+  "We are going to send you an email containing a link to reset your password": "You will receive an e-mail with link to reset your password",
   "Cancel": "Cancel",
-  "Information are missing, please use the link in the email your should have received to reset your password": "Information are missing, please use the link in the email your should have received to reset your password",
-  "Your password has been changed, you can now login": "Your password has been changed, you can now login",
+  "Information are missing, please use the link in the email your should have received to reset your password": "Info missing. Please use the link in the e-mail your should have received to reset your password",
```

Comment:
Should we abbreviate Information ?

Reply:
I don't know about the abbreviation, but information is uncountable and cannot be used with "are" (there is a mistake in the key).

Reply:
I changed to "Information" in the translation and kept the source key for now.